crates/pop-cli/src/commands/clean.rs

```diff
@@ -33,7 +40,7 @@ pub(crate) struct CleanCacheCommand<'a, CLI: Cli> {
 
 impl<'a, CLI: Cli> CleanCacheCommand<'a, CLI> {
 	/// Executes the command.
-	pub(crate) fn execute(self) -> Result<()> {
+	pub(crate) fn execute(self, args: CleanCommandArgs) -> Result<()> {
```

Comment:
To maintain consistency with the other commands that only have `execute(self)`, add the add flag `all` in the `CleanCacheCommand` struct and then pass it in the `commands/mod.rs` instead of adding it here.
```
Self::Clean(args) => match args.command {
  clean::Command::Cache(cmd_args) => {
	  // Initialize command and execute
	  clean::CleanCacheCommand { cli: &mut Cli, cache: cache()?, all: cmd_args.all }
		  .execute()
		  .map(|_| Value::Null)
  },
  },
```
So you can use it inside `execute` with `self.all`.

Reply:
I see! That definitely makes sense to do.